spec/views/profiles/show.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "profiles/show.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
I'm guessing one of the generators made these. We haven't covered view tests and they're sort of pointless IMO. `git rm` !